lib/services/orphaned_packages_check.pm

```diff
@@ -0,0 +1,38 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017-2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Check for any orphaned packages. There should be none in fully
+# supported systems
+# Maintainer: Oliver Kurz <okurz@suse.com>
+# Tags: poo#19606
+
+package services::orphaned_packages_check;
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils 'zypper_call';
+use version_utils 'is_upgrade';
+
+sub orphaned_packages_list {
+    my $cmd = 'zypper pa --orphaned | grep -v "\(release-DVD\|release-dvd\)" | (! grep "@System")';
```

Comment:
this is reverting the latest change to also ignore the openSUSE live media packages

Reply:
Ok, thanks for your reminder. I'll take a look at this and verify more cases.

Reply:
What I meant is that probably you did a mistake in conflict resolution when rebasing your local git branch. https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/9186 shows the change that is missing in your PR.